webpack_loader/utils.py

```diff
@@ -7,11 +8,18 @@
 
 
 def get_loader(config_name):
-    if config_name not in _loaders:
-        config = load_config(config_name)
+    config = load_config(config_name)
+
+    loader_name = config_name
+    locale = None
+    if config["LOCALIZED_BUILDS"]:
+        locale = translation.get_language()
+        loader_name += "-" + locale
```

Comment:
Is it possible to somehow support without a new config var? It'd be ideal if django side has as little configuration as possible and we push down the configuration to JS side whenever we can.

Reply:
If that is really necessary, one could guess a user locale (e.g. "en"), that wouldn't affect the functionality otherwise.
However, I think the better way to do this is by simply leaving the configuration. There are actually quite a few extra variables, see the config.py.